doc/nrf/libraries/networking/azure_iot_hub.rst

```diff
@@ -91,16 +91,16 @@ To connect to Azure IoT Hub without using DPS, complete the following minimum re
 1. In the `Azure Portal`_, navigate to :guilabel:`IoT Hub` and select the desired IoT hub.
 #. In the overview page, locate and copy the ``Hostname`` and configure :kconfig:option:`CONFIG_AZURE_IOT_HUB_HOSTNAME` to this address.
 
-   The hostname can also be set at run time.
-#. Set the option :kconfig:option:`CONFIG_AZURE_IOT_HUB_DEVICE_ID` to the device ID.
+   You can also set the hostname at run time.
+#. Set the Kconfig option :kconfig:option:`CONFIG_AZURE_IOT_HUB_DEVICE_ID` to the device ID.
 
    The device ID must match with the one used while creating the certificates.
    You can also set the device ID at run time by populating the ``device_id`` member of the :c:struct:`azure_iot_hub_config` structure passed to the :c:func:`azure_iot_hub_connect` function when connecting.
    If the ``device_id.size`` buffer size is zero, the compile-time option :kconfig:option:`CONFIG_AZURE_IOT_HUB_DEVICE_ID`` is used.
 #. Make sure that the device is already registered with your Azure IoT Hub, or follow the instructions in `Registering the device with Azure IoT Hub`_.
-#. Set the :kconfig:option:`CONFIG_MQTT_HELPER_SEC_TAG` option to the security tag used in :ref:`azure_iot_hub_flash_certs`.
+#. Set the :kconfig:option:`CONFIG_MQTT_HELPER_SEC_TAG` Kconfig option to the security tag used in :ref:`azure_iot_hub_flash_certs`.
 
-   Optionally, set the :kconfig:option:`CONFIG_MQTT_HELPER_SEC_TAG` option if multiple server certificates are provisioned.
+   Optionally, set the :kconfig:option:`CONFIG_MQTT_HELPER_SEC_TAG` Kconfig option if multiple server certificates are provisioned.
```

Comment:
There's a pre-existing typo here that is fixed in #10813 (https://github.com/nrfconnect/sdk-nrf/pull/10813/files#r1165247589), but might as well be done here.

```suggestion
   Optionally, set the :kconfig:option:`CONFIG_MQTT_HELPER_SECONDARY_SEC_TAG` Kconfig option if multiple server certificates are provisioned.
```